api/extension-guides/chat.md

```diff
@@ -269,6 +269,77 @@ cat.followupProvider = {
 
 > **Tip:** Follow-ups should be written as questions or directions, not just concise commands.
 
+### Implement participant detection
+
+To make it easier to use chat participants with natural language, you can implement participant detection. Participant detection is a way to automatically route the user's question to a suitable participant, without having to explicitly mention the participant in the prompt. For example, if the user asks "How do I optimizer this query?", the question would be automatically routed to the `@database` participant.
+
+To implement participant detection, you specify the `disambiguation` property in the extension `package.json` file. The `disambiguation` property contains a list of detection categories, each with a description and examples.
```

Comment:
Maybe add a sentence "Description and examples will be used by VS Code to determine if the user query should be sent to your participant"

Reply:
Added